app/javascript/react/pages/NotFoundPage/NotFoundPage.style.tsx

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+const NotFoundContainer = styled.div`
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 50vh;
+`;
+const NotFoundContent = styled.div`
```

Comment:
Same here, what about `ContentContainer`? It is imported as NotFoundPage.style and I think it says enough about it's purpose, so we could shorten the names. It's up to you, just suggesting 😄